lib/src/analyzers/unused_l10n_analyzer/unused_l10n_visitor.dart

```diff
@@ -117,4 +127,22 @@ class UnusedL10nVisitor extends RecursiveAstVisitor<void> {
       );
     }
   }
+
+  void _addMemberInvocationOnAccessor(SimpleIdentifier target, String name) {
+    final staticElement =
+        target.staticElement?.enclosingElement as ExtensionElement;
+    for (final element in staticElement.accessors) {
+      if (_classPattern.hasMatch(element.name) && element.isGetter) {
```

Comment:
This is the place: you basically check for a getter and match. Instead, I'd suggest to check the return value of a getter for a patter match. That would fit into the whole command idea.

Reply:
Yeah, the problem is my file (that contains the extension) wasn't included when I was using -p `DemoLocalizations`.

Reply:
With the current code?

Reply:
I think you are right, not sure why I was getting different results, I'll check my code again and I'll send some changes :D thanks

Reply:
@incendial I sent my changes , can you check please? do you have any hints about the test? I'm using `dart test`.